```
Processing ./diagrams-0.21.1-py3-none-any.whl
ERROR: Could not find a version that satisfies the requirement graphviz<0.20.0,>=0.13.2 (from diagrams) (from versions: none)
ERROR: No matching distribution found for graphviz<0.20.0,>=0.13.
```

---

```
3 errors generated.
make: *** [../../make/python3.CLANG7-LINUX-X86_64.makefile:9962: /build/source/catboost/python-package/build/temp.linux-x86_64-3.10/catboost/python-package/catboost/_catboost.pyx.cpp.pic.o] Error 1
make: *** Waiting for unfinished jobs....
Traceback (most recent call last):
  File "/build/source/catboost/python-package/nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 260, in <module>
    setup(
  File "/nix/store/rqpafq1g9r7k45bzfjn8jsb6vssyr2dh-python3.10-setuptools-61.2.0/lib/python3.10/site-packages/setuptools/__init__.py", line 87, in setup
    return distutils.core.setup(**attrs)
  File "/nix/store/rqpafq1g9r7k45bzfjn8jsb6vssyr2dh-python3.10-setuptools-61.2.0/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 148, in setup
    return run_commands(dist)
  File "/nix/store/rqpafq1g9r7k45bzfjn8jsb6vssyr2dh-python3.10-setuptools-61.2.0/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 163, in run_commands
    dist.run_commands()
  File "/nix/store/rqpafq1g9r7k45bzfjn8jsb6vssyr2dh-python3.10-setuptools-61.2.0/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 967, in run_commands
    self.run_command(cmd)
  File "/nix/store/rqpafq1g9r7k45bzfjn8jsb6vssyr2dh-python3.10-setuptools-61.2.0/lib/python3.10/site-packages/setuptools/dist.py", line 1214, in run_command
    super().run_command(command)
  File "/nix/store/rqpafq1g9r7k45bzfjn8jsb6vssyr2dh-python3.10-setuptools-61.2.0/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 986, in run_command
    cmd_obj.run()
  File "setup.py", line 190, in run
    self.build_with_make(topsrc_dir, build_dir, catboost_ext, put_dir, verbose, dry_run)
  File "setup.py", line 236, in build_with_make
    logging_execute(make_cmd, verbose, dry_run)
  File "setup.py", line 62, in logging_execute
    subprocess.check_call(cmd, universal_newlines=True)
  File "/nix/store/rl63y9f2bys4zd558dlkrj7ycj1x4w62-python3-3.10.4/lib/python3.10/subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make', '-f', '../../make/python3.CLANG7-LINUX-X86_64.makefile', 'CC=clang', 'CXX=clang++', 'PYTHON=python2', 'BUILD_ROOT=/build/source/catboost/python-package/build/temp.linux-x86_64-3.10', 'SOURCE_ROOT=/build/source/catboost/python-package/../..', '-j', '128']' returned non-zero exit status 2.
```
Not sure why we have python2 in here.

---

--- I can run another nixpkgs-review later. The current didn't catch many logs.

--- a/pkgs/development/python-modules/diagrams/default.nix
+++ b/pkgs/development/python-modules/diagrams/default.nix
@@ -27,8 +27,7 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace 'jinja2 = "^2.10"' 'jinja2 = "*"' \
-      --replace 'graphviz = ">=0.13.2,<0.17.0"' 'graphviz = "*"'
+      --replace 'graphviz = ">=0.13.2,<0.20.0"' 'graphviz = "*"'
   '';
 
   preConfigure = ''

